torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -85,6 +85,14 @@ class TORCH_API TensorExprKernel {
     return graph_;
   }
 
+  const std::vector<ConstantDescr>& getConstantDescriptors() const {
```

Comment:
@ZolotukhinM in the current prototype we serialize the constant parameters as IValue wrapped tensors in the compiled model file: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/mobile/nnc/context.h#L131

Looks like ConstantDescr contains Buf which has dims (represented as Expr - they are constant, right?) and dtype - so it's possible to reconstruct the tensors and serialize, right? Or shall we directly wrap the ConstantDescr and the raw bytes into IValue?

Another option is to expose a way to inspect the link to the original IR node. It might allow us to persist things represented in the form of opaque tensors used by external ops (for which NNC generated assembly could simply pass through).

Reply:
> ConstantDescr contains Buf which has dims (represented as Expr - they are constant, right?)

This is a more general case and Exprs do not have to always be constant. However, in our case we can add an assert that they are and then convert to `vector<int>` to construct `InputDesc` or something similar if we need it. When the shapes are static we expect this exprs to be constant. When we extend to support dynamic shapes that assert would need to be relaxed, but the interface to query the info about tensors would stay the same and we will already have a way to represent symbolic shapes (with Var expressions).